torch/csrc/jit/passes/constant_propagation.cpp

```diff
@@ -129,6 +135,11 @@ struct ConstantPropagator {
         // forwards tuples later in the graph, such as a Tuple index
         auto tuple_val = n->outputs()[i];
         if (auto new_tup = tryInsertTuple(outputs[i], tuple_val)) {
+          GRAPH_UPDATE(
+              "Folding tuple\n",
+              *(n->outputs()[i]->node()),
```

Comment:
I think we should use the value, not the node here. Otherwise we might have several log-messages referring to the same node, like:
```
Folding tuple
%a, %b = my::op(...)
to
with
"%a_new = my::other_op()

Folding tuple
%a, %b = my::op(...)
to
with
"%b_new = my::yet_another_op()
```
which could be confusing.

Reply:
Agreed